pkg/database/fake/database_generated.go

```diff
@@ -73,15 +73,15 @@ func (f *Database) queryResources(_ context.Context, kind, version, _, _ string,
 	return f.filterResourcesByKindAndApiVersion(kind, version)
 }
 
-func (f *Database) QueryLogURLs(ctx context.Context, kind, apiVersion, namespace, name string) ([]string, error) {
+func (f *Database) QueryLogURL(ctx context.Context, kind, _, _, _ string) (string, error) {
 	if kind == "Pod" {
-		return []string{f.logUrl[0].Url}, f.err
+		return f.logUrl[0].Url, f.err
```

Comment:
Why is `Pod` differentiated here?

Reply:
Right, with the latest changes this now doesn't make sense, I need to just return `f.logUrl[0]`

Reply:
fixed